hamocc/mo_carchm.F90

```diff
@@ -254,8 +257,8 @@ subroutine carchm(kpie,kpje,kpke,kbnd,pdlxp,pdlyp,pddpo,prho,pglat,omask,psicomo
               natco3(i,j,k) = natcc * rrho
             endif
 
-            ! solubility of O2 (Weiss, R.F. 1970, Deep-Sea Res., 17, 721-735) for moist air
-            ! at 1 atm; multiplication with oxyco converts to kmol/m^3/atm
+            ! solubility of O2 (Weiss, R.F. 1970, Deep-Sea Res., 17, 721-735) for moist air at
+            ! 1 atm; multiplication with oxyco converts to kmol/m^3/atm, temp=[-1,40], saln=[0,40]
```

Comment:
here and below: I would suggest to write `temp_min`,`temp_max` , similar for saln (or at least correct the values below: -2 -> -1, 30 -> 40, ...)

Reply:
The temperature and salinity ranges given in comments indicate the valid range of the fit. This is usually the range for which measurements have been available, taken from the original publication. I have added a comment in carchm_solve to clarify this